src/toimg.c

```diff
@@ -166,7 +166,7 @@ int main_toimg(int argc, char* argv[argc])
 
 	num_init();
 
-	char* ext = rindex(argv[2], '.');
+	char* ext = strrchr(argv[2], '.');
```

Comment:
While `rindex` is a deprecated POSIX function and not implemented in MinGW, `strrchr` is is part of the C standard library and is cross-platform.